docs/design/features/native-hosting.md

```diff
@@ -316,7 +384,7 @@ Closes a host context.
 * `host_context_handle` - handle to the initialized host context to close.
 
 
-### Loading managed components
+### Loading and calling managed components
```

Comment:
The description of `hdt_load_assembly_and_get_function_pointer` above was referring to this section's name and should be updated as well.